### ISSUE TITLE
sie-front

### ISSUE THREAD

**Issue by zaiosc-dpers** [DefensoriaPublicaRS/sie-front#13]

**Comment by hbribeiro77** [comment]
Mostrar isso aqui pro usuário na lista de intimações (ou pelo menos no arquivo que se vai baixar para abrir no excel e assemelhados):
**Número do processo**,  **Intimado**, **Evento**(identificadorMovimento), **data/hora da expedição da intimação**(dataHoraMovimento), **Descrição**(descricaoMovimento), **Situação**(status), **Prazo,** **Data inicial**(prazoinicioPrazo), **Data final**(finalPrazo), **Órgão julgador**(nomeOrgao), **Código do município**(codigoMunicipioIBGE)


Observação, quanto ao código do município, o **desejável** seria mostrarmos o nome do município e não seu código. Aparentemente o IBGE tem serviço que faz isso: https://servicodados.ibge.gov.br/api/v1/localidades/municipios/4300604


![Image](https://github.com/DefensoriaPublicaRS/sie-front/assets/39736227/84772fb0-e591-4214-bda2-2269679300e1)

**Comment by marcelogm** [comment]
> Observação, quanto ao código do município, o **desejável** seria mostrarmos o nome do município e não seu código. Aparentemente o IBGE tem serviço que faz isso: https://servicodados.ibge.gov.br/api/v1/localidades/municipios/4300604
]exeqAw2BovnjoqyR_STw)

Podemos consultar ou fazer um cache disso.

Qual a vantagem de mostrar o município? Pergunto porque até onde eu lembro, o nome do orgão julgador já tem o município dentro. Ou estou enganado? 🤔 

@hbribeiro77 @zaiosc-dpers

**Comment by hbribeiro77** [comment]
Não sei se vai ter sempre o nome do município no OJ, principalmente naqueles muitos específicos, e para filtrar fica bem fácil pro usuário filtrar pela suas cidades atendidas.

**Comment by marcelogm** [comment]
> Não sei se vai ter sempre o nome do município no OJ, principalmente naqueles muitos específicos, e para filtrar fica bem fácil pro usuário filtrar pela suas cidades atendidas.

@hbribeiro77 certo, essa do muncípio podemos transformar numa issue a parte então, porque vamos ter que buscar essa info e não só exibir na tabela. Pode criar a issue pra gente?

**Comment by zaiosc-dpers** [comment]
Sugestão de layout:

![image](https://github.com/DefensoriaPublicaRS/sie-front/assets/169402696/e06c9b47-9a77-4495-8aeb-6be1851a2d65)

OBS.: Quando for possível buscar e mostrar o nome do município, o label da coluna ficaria apenas "Município".

**Comment by zaiosc-dpers** [comment]
Verificando em Testes...

**Comment by zaiosc-dpers** [comment]
@marcelogm Opa, ainda não temos uma coluna "Prazo" com o número de dias `payload.outros.prazo` e também falta tratar os valores do status como é apresentado no Portal (ver abaixo) (conforme sugestões em https://github.com/DefensoriaPublicaRS/sie-front/issues/13#issuecomment-2105288962).

- Prazo: atualmente aparecem apenas as datas, mas não o prazo em dias

![image](https://github.com/DefensoriaPublicaRS/sie-front/assets/169402696/5173b05e-cf18-4d22-928d-cc70385214d1)

![image](https://github.com/DefensoriaPublicaRS/sie-front/assets/169402696/fc9daeb7-3829-4378-b273-aa9e9004aee5)


- Status: `payload.outros.status`

XML | Portal 👇 
|:----|:----|
Aguardando Abertura | Pendente
Aberto | Em curso

**Comment by zaiosc-dpers** [comment]
Os ajustes sugeridos foram incluídos em https://github.com/DefensoriaPublicaRS/sie-front/issues/40.
